pytissueoptics/rayscattering/fresnel.py

```diff
@@ -0,0 +1,102 @@
+import math
+import random
+from dataclasses import dataclass
+
+from pytissueoptics.scene import Vector, Material
+from pytissueoptics.scene.intersection.intersectionFinder import Intersection
+
+
+@dataclass
+class FresnelIntersection:
+    nextMaterial: Material
+    incidencePlane: Vector
+    isReflected: bool
+    angleDeflection: float
+
+
+class FresnelIntersectionFactory:
```

Comment:
Factory is very intimidating and might be changed. "compute" might as well be changed to "computeIntersection" ?

Reply:
Let's say FresnelIntersect, like previously named, though it is now very close to the FresnelIntersection dto. 
Renaming compute to computeIntersection will sound like it is creating Intersection dtos instead of FresnelIntersection. And it would also sound not so great when called -> `fresnelIntersect.computeIntersection(rayDirection, intersection)`